bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Display.java

```diff
@@ -6832,4 +6832,30 @@ static long windowProc(long id, long sel, long arg0, long arg1, long arg2, long
 static boolean isActivateShellOnForceFocus() {
 	return "true".equals(System.getProperty("org.eclipse.swt.internal.activateShellOnForceFocus", "true")); //$NON-NLS-1$
 }
+
+/**
+ * {@return whether rescaling of shells at runtime when the DPI scaling of a
+ * shell's monitor changes is activated for this device}
+ *
+ * @since 3.127
+ */
+public boolean isRescalingAtRuntime() {
+	return false;
+}
+
+/**
+ * Activate or deactivates rescaling of shells at runtime whenever the DPI
+ * scaling of the shell's monitor changes. This is only safe to call as long a
+ * no shell has been created for this display. When changing the value after a
+ * shell has been created for this display, the effect is undefined.
+ * <p>
+ * <b>Note:</b> This functionality is only available on Windows. Calling this
+ * method on other operating system will have no effect.
+ *
+ * @param activate whether rescaling shell be activated or deactivated
```

Comment:
I would say either 'sh**a**ll' or 'rescaling of shells' 🤔

Reply:
It was supposed to be "shall". I just mixed up the two words 😉 
Fixed with 784ffc7.